roofit/roofit/src/RooPoisson.cxx

```diff
@@ -65,8 +66,11 @@ RooPoisson::RooPoisson(const char *name, const char *title,
 Double_t RooPoisson::evaluate() const
 {
   Double_t k = _noRounding ? x : floor(x);
-  if(_protectNegative && mean<0)
-    return 1e-3;
+  if(_protectNegative && mean<0) {
```

Comment:
is _protecNegative set by default i RooPoisson ?

Reply:
It's actually off by default. I will enable it in an extra commit.